packages/components/src/ui/styles/index.js

```diff
@@ -0,0 +1,10 @@
+export * from './core';
```

Comment:
While it's convenient to re-export everything, I think we need to have better control over what goes into the public API in the future. I noticed this pattern used quite often in the `ui` folder. The biggest issue I foresee is that new methods can get exported only to use with tests but through the chain of catch it all re-exports they might suddenly become available for usage outside of the package.